common/utils.js

```diff
@@ -2630,6 +2630,27 @@
             return defer.promise;
         }
 
+        /* Custom function to add styling based on browser type and operating system */
+        function addNewClassForBrowerInOs(){
+          var osName = "unidentified";
+          var browserName = "unidentified";
+
+          if ((navigator.platform.indexOf("Mac")!= -1) && (navigator.userAgent.indexOf("Firefox")!=-1))
+          {
+            osName="MacOS";
+            browserName = "Firefox";
+          }
+
+          if((osName === "MacOS") && (browserName === "Firefox"))
+          {
+              var newCustomClass = "chaise-mac-firefox";
```

Comment:
Instead of forcing the conditional check to be for BOTH mac and firefox, it's better if we do each check separately. We want to add a separate class for both the OS and the browser. So in the case that we had an issue with, we want to add 2 classes to the element, `chaise-mac` and `chaise-firefox`.

Let's change this whole function to something like the following:
```
function addMacFirefoxClass () {
  var osClass = ( navigator.platform.indexOf("Mac") != -1 ? "chaise-mac" : undefined );
  var browserClass = ( navigator.userAgent.indexOf("Firefox") != -1 ? "chaise-firefox" : undefined );

  var bodyElement = document.querySelector(".chaise-body");
  if (bodyElement){
    if (osClass) UiUtils.addClass(bodyElement, osClass);
    if (browserClass) UiUtils.addClass(bodyElement, browserClass);
  }
}
```

After making this change, you will have to update your selector in the css document as well to `.chaise-mac.chaise-firefox`.

We want 2 separate classes here so we can later expand this for each OS and browser type and more easily define combination of rules and classes without iterating over every combination.